### DISCUSSION TITLE
Is there a discord?

### DISCUSSION THREAD

**Discussion by Draqueus** [phst-randomizer/ph-randomizer#163] [Q&A] [answered]
This was my favorite Zelda game when it came out, and I am SO THRILLED for this to be in active development! I have absolutely no coding skill, so I can't help there, but I'd love to help with any in-game testing once any alpha or beta builds come out. Is there a discord or anything set up for discussions on this rando?

**Comment by lunarsoap5** [comment, marked as answer]
While there technically is a discord, it is currently private and used for development discussion. The randomizer is still *very* early in development and we are still a fair bit away from any alpha or beta testing. As time goes on and we progress further in development, we may open up the discord for further discussion and collaboration.

**Reply by Draqueus** [reply]
Sounds great! I'll be watching the updates here with great interest :)
